lib/workers/repository/updates/branch-name.ts

```diff
@@ -47,6 +47,8 @@ function cleanBranchName(
 
 export function generateBranchName(update: RenovateConfig): void {
   // Check whether to use a group name
+  const newMajor = String(update.newMajor);
+  const newMinor = String(update.newMinor);
```

Comment:
```suggestion
  const newMajor = `${update.newMajor}`;
  const newMinor = `${update.newMinor}`;
```
use string template instead of String function.

Reply:
accidentally resolved 🙈